api/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.0
 	github.com/gravitational/trace v1.2.1
-	github.com/jonboulle/clockwork v0.3.0
+	github.com/jonboulle/clockwork v0.4.0
```

Comment:
Mentioned in other update, but I'd keep an eye out for This might cause problems: https://github.com/jonboulle/clockwork/pull/55.

It may be best to pull this into its own PR, so it's easy to rollback and/or do any changes related to it.